components/scripts/lib/build-scan-online.sh

```diff
@@ -53,28 +53,27 @@ fetch_single_build_scan() {
   parse_single_build_scan "${build_scan_data}"
 }
 
+# The value of logging_level should be either 'brief_logging' or
+# 'verbose_logging'
```

Comment:
💡In the future, we could rename these to just "brief" and "verbose". Including "logging" here is arguably redundant.   We could also update the fetch tool to support this with an actual CLI parameter rather than reading it from a positional parameter.

Thinking about it, "quiet" might be a better term. "quiet" is used more often than "brief" in other applications for this sort of thing.

Reply:
I actually did consider going with just `brief` and `verbose` but I was afraid it would not be clear enough about what it meant. If you just see `verbose` at the call site, it's not clear what is supposed to be verbose without digging into the function's code. For example:

```bash
fetch_build_scans_and_build_time_metrics 'verbose' "${build_scan_urls[@]}"
```

Then again, it is common for command line tools to have `--quiet` and `--verbose` flags and it is usually clear what it means. So maybe I am overthinking it. I would be fine to change it to how you describe if you think that it is clear enough. 

> We could also update the fetch tool to support this with an actual CLI parameter rather than reading it from a positional parameter

Unless I misunderstand what you mean, this is already the case.

https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/97caa96500593310919013e34df9f61e30523df5/components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildScanDataCommand.java#L52-L53

> Thinking about it, "quiet" might be a better term. "quiet" is used more often than "brief" in other applications for this sort of thing.

Ha, I do think it is more common, but usually quiet means no output. In this case we still output some details, just not as much.

Reply:
I would stick to the current values. They are a bit more expressive.